easyaudit/signals/model_signals.py

```diff
@@ -62,14 +62,16 @@ def pre_save(sender, instance, raw, using, update_fields, **kwargs):
                 # We need a better way for this to work. ManyToMany will fail on pre_save on create
                 return None
 
-            if instance.pk is None:
```

Comment:
I wonder if there are performance considerations here.

I've made a similar change in a PR but I opted to keep the `if instance.pk`. My logic was that if `pk` is None it's reasonable to assume that the object does not exist in database, this way a database request can be saved.

I don't know what way is best, hopefully more experienced people than me can chime in.

Reply:
I tend to agree and i think your approach is better than this. either way the performance is still impacted with a query.

Reply:
@Etenil Added your changes from #141